### ISSUE TITLE
riak debug broken in Riak KV 3.0.1

### ISSUE THREAD

**Issue by Bob-The-Marauder** [basho/riak#1037]
**Expected behaviour KV 3.0.1:**
Execute `riak debug` and a debug.tar.gz file is created containing lots of information about the node.

**Actual behaviour KV 3.0.1**
```
root@buster:~# riak debug
Required file app_epath.sh not found. Expected here:
/usr/lib/riak/lib/app_epath.sh
See 'riak-debug -h' and manpage for help.
```

This has been tested with CentOS8, Debian Buster and Ubuntu Focal and Bionic with the same result for all of them.

**Comment by martinsumner** [comment]
app_epath.sh used to be provided as part of [node_package](https://github.com/basho/node_package/blob/develop-3.0/priv/base/app_epath.sh), but we moved away from node_package to use relx.

If you want to get this working (or at least past this stage), you will need to add the script back into rel/files and add into rebar.config a relx overlay template to copy the script across to the desired location like https://github.com/basho/riak/blob/develop-3.0/rebar.config#L83-L95.  Use copy if there's no substitution required of standard variables, template if there is.

We came across riak debug not working during testing of 3.0, but as we don't use it, we didn't spend time fixing it.  Not sure problems there might be beyond the existence of this script, so how much work there would be involved in fixing it - but if it is just about correcting the existence of the script it should be straight forward.
